crates/ruff/src/rules/pycodestyle/rules/mod.rs

```diff
@@ -86,3 +86,57 @@ mod whitespace_around_keywords;
 mod whitespace_around_named_parameter_equals;
 mod whitespace_before_comment;
 mod whitespace_before_parameters;
+
+#[allow(unused)]
+enum Whitespace {
```

Comment:
Wonder whether we should find another home for this? Maybe `pycodestyle/whitespace.rs`? Or `pycodestyle/rules/helpers.rs`?

Reply:
I can put it into `whitespace` and potentially gate it to the feature. 

I prefer not to put code into `helpers`/`util` modules because they often end up being this huge pile of random things that you have a hard time finding in the future. I prefer to come up with a concrete concept where the code fits in.

Reply:
Embrace the helpers...

Reply:
I'll move this into `LogicalLines` in my top most PR (easier than dealing with all the merge conflicts). The idea is that the `LogicalLine` already has a `before_text` and I think it's only used to determine the Whitespace... so I'll add it there.